src/main/java/duke/logic/command/order/DeleteOrderCommand.java

```diff
@@ -24,7 +24,7 @@
     private final Set<Index> indices;
 
     /**
-     * Creates a {@code DeleteOrderCommand}.
+     * Creates a {@code DeleteProductCommand}.
```

Comment:
Why the change?

Reply:
Sorry I think I accidentally changed it...